src/api/src/services/BinaryFlashingStrategy/index.ts

```diff
@@ -577,6 +577,7 @@ export default class BinaryFlashingStrategyService implements FlashingStrategy {
           params
         );
       }
+      console.log({ flasherArgs });
```

Comment:
I believe this `console.log` statement has sneaked into this PR while you were still developing this feature. The expected way of logging in API backend is with the logger class,

Reply:
Yup, removed.